tsconfig.json

```diff
@@ -19,7 +19,17 @@
     "downlevelIteration": true,
     "allowJs": true,
     "useDefineForClassFields": false,
-    "lib": ["es2015", "es2016", "es2017", "es2019", "esnext.string", "esnext.array", "esnext.asynciterable", "dom"],
+    "lib": [
+      "es2015",
+      "es2016",
+      "es2017",
+      "es2019",
+      "es2021",
+      "esnext.string",
+      "esnext.array",
+      "esnext.asynciterable",
+      "dom"
+    ],
```

Comment:
FYI the dev server compiles fine with `["ES2022", "dom"]` if we switch the target to ES2022.